.github/workflows/build-images.yml

```
@@ -15,7 +15,7 @@ jobs:
       uses:                       redhat-actions/buildah-build@v2
```

```suggestion
      uses:                       redhat-actions/buildah-build@7a95fa7ee0f02d552a32753e7414641a04307056 # v2.13
```
yes, latest version of the action :)